build.sbt

```diff
@@ -161,15 +73,16 @@ val root = project.in(file("."))
     .settings(commonSettings: _*)
     .settings(
       libraryDependencies ++= dep,
-      verifyDependencies in verify ++= verifyDeps,
+      verifyOutputFile in verifyGenerate := baseDirectory.value / "verify.sbt",
       verifyOptions in verify := VerifyOptions(
         includeBin = true,
         includeScala = true,
         includeDependency = true,
         excludedJars = Nil,
-        warnOnUnverifiedFiles = true,
-        warnOnUnusedVerifications = true
+        warnOnUnverifiedFiles = false,
+        warnOnUnusedVerifications = false
```

Comment:
why this change? it seems keeping it true is more secure

Reply:
`false` means error instead of warning. I actually didn't plan on changing this, but it happened during some fixes I applied wrt `sbt-verify`. 

I'm not having any problems with `sbt-verify` now, so I wouldn't mind staying with `false`. WDYT?

Reply:
Makes sense, I didn't know it means an error :)